instrumentation/spring-aop-2/build.gradle

```diff
@@ -4,9 +4,6 @@ dependencies {
     implementation("org.springframework:spring-aop:3.2.+")
     //     implementation("org.springframework:spring:2.0.3") // First version to have MethodMatchers.matches()
 
-    testImplementation("org.powermock:powermock-module-junit4-rule-agent:1.6.1")
-    testImplementation("org.powermock:powermock-module-junit4:1.6.1")
-    testImplementation("org.powermock:powermock-api-mockito:1.6.1")
```

Comment:
How is this related to this PR?

Reply:
I was looking for mocking static methods. Found these and noticed that this module had no tests.